airflow/models/baseoperator.py

```diff
@@ -1121,6 +1139,12 @@ def set_upstream(self, task_or_task_list: Union['BaseOperator', List['BaseOperat
         """
         self._set_relatives(task_or_task_list, upstream=True)
 
+    @property
+    def output(self):
+        """Returns default XComArg for the operator"""
```

Comment:
Can we add a bit more detail, something like Output from the previous task using XComArg or something similar so that is simpler to understand. WDYT ?

Reply:
"XCom representation of output received from this operator"?

Reply:
"Returns reference to XCom pushed by current operator"?

Reply:
Either of them is fine :)

Reply:
We haven't updated this yet :)

Reply:
Merging this PR ignoring this change (we can update it in a related follow-up PR)

Reply:
Addressed in https://github.com/apache/airflow/pull/8805